contentcuration/contentcuration/models.py

```diff
@@ -2594,11 +2594,12 @@ class CommunityLibrarySubmission(models.Model):
     )
     categories = models.JSONField(blank=True, null=True)
     date_created = models.DateTimeField(auto_now_add=True)
-    date_resolved = models.DateTimeField(blank=True, null=True)
+    date_updated = models.DateTimeField(blank=True, null=True, db_index=True)
```

Comment:
Here, we also need an `auto_now=True` argument so that this field gets updated automatically every time an instance is updated